app/views/groups/members/_current_user_is_member.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= viral_dropdown(icon: "ellipsis_vertical",
```

Comment:
**question (non-blocking):** Gitlab does not use dropdowns for these, I am assuming because it is a single action and is partially hidden in the dropdown.  Should we move these out to buttons on the table row?  Thoughts @ericenns